### PR TITLE
FW-5851-feather-logo

### PR DESCRIPTION
**Pull request by Cara-Barter**
Description of Changes

- replaced png and svg images and sized header image

Checklist

- [ ] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- [ ] Tests have been updated / created if applicable

Reviewers Should Do The Following:

- [x] Review the changes here
- [ ] Pull the branch and test locally

Additional Notes
N/A

### CODE REVIEW

**Review by gmcauliffe — CHANGES_REQUESTED** [review]
These replacement logo images need to maintain a similar ratio of logo to background as the old images. Also we need to avoid distorting the aspect ratio of the logo itself when it is resized, for branding consistency. 

@sarahfirstvoices is this something Comms could assist us with?

File: public/default/logo512.png
Comment (sarahfirstvoices) [inline public/default/logo512.png:1]: Looks like these smaller ones have been warped, compared to the original. Do you need help scaling them with the same aspect ratio?

**Review by sarahfirstvoices — CHANGES_REQUESTED** [review]
We'll need to fix the aspect ratio for the images in the default folder

**Comment by sarahfirstvoices** [comment]
> @sarahfirstvoices is this something Comms could assist us with?

Mind meld. Yes probably

**Review by sarahfirstvoices — APPROVED** [review]

**Review by gmcauliffe — APPROVED** [review]

**Pull request merged**